### PR TITLE
revise README to have consistent information

### PR DESCRIPTION
**Pull request by ev1stensberg**
I tried this approach, as how I understood how it works. Copied a section from the v.18 changelog, so hopefully that is correct :) #609  relative

 - [ ] CHANGELOG.md has been updated

### CODE REVIEW

File: src/init/element/templates/polymer-2.x/README.md [hunk 1]
```diff
@@ -18,9 +18,8 @@ $ polymer serve
 $ polymer build
 ```
 
-This will create a `build/` folder with `bundled/` and `unbundled/` sub-folders
-containing a bundled (Vulcanized) and unbundled builds, both run through HTML,
-CSS, and JS optimizers.
+This will create a `build/` folder with `default/` and `bower_components/` sub-folders
```
Comment (justinfagnani) [inline src/init/element/templates/polymer-2.x/README.md]: it shouldn't produce a `bower_components/` folder outside of the individual build folders.

File: src/init/element/templates/polymer-2.x/README.md [hunk 2]
```diff
@@ -18,9 +18,8 @@ $ polymer serve
...
+containing a unbundled build, run through HTML, CSS, and JS optimizers. New build options have been added to give you more control over the generated build. These options can be defined in your project's `polymer.json`, or via CLI flags. Run `polymer build --help` to see a list of new supported CLI flags.
```
Comment (justinfagnani) [inline src/init/element/templates/polymer-2.x/README.md]: whether it's bundled or unbundled is determined by the build options

**Comment by ev1stensberg** [comment]
This seems to me a bug that's not related to the README. Could you give me some info on how it should look in v2 and the default behaviour? I'll be happy to rephrase

**Comment by ev1stensberg** [comment]
If there's no update on this, I'm going to go ahead and close this by the end of the week :)

**Comment by justinfagnani** [comment]
Hey @ev1stensberg, I'm just getting back from vacation today, give me a couple of days.

**Comment by ev1stensberg** [comment]
@justinfagnani That's fine, speak soon!

**Comment by FredKSchott** [comment]
Thanks for the contribution @ev1stensberg! All of the templates have README issues, so we're going to fix them all at once in #656

**Pull request closed without merging**